Source/SharpDX.Direct3D12/GraphicsCommandList.cs

```diff
@@ -204,56 +205,51 @@ public unsafe void SetRenderTargets(CpuDescriptorHandle? renderTargetDescriptor,
             SetRenderTargets(renderTargetDesc.Ptr != PointerSize.Zero ? 1 : 0, renderTargetDescriptor.HasValue ? new IntPtr(&renderTargetDesc) : IntPtr.Zero, false, depthStencilDescriptorRef);
         }
 
-        /// <summary>	
-        /// <p>Sets a CPU descriptor handle for the vertex buffers.</p>	
-        /// </summary>	
-        /// <param name="startSlot"><dd>  <p> Index into the device's zero-based array to begin setting vertex buffers. </p> </dd></param>	
-        /// <param name="numViews"><dd>  <p> The number of views in the <em>pViews</em> array. </p> </dd></param>	
-        /// <param name="viewsRef"><dd>  <p> Specifies the vertex buffer views in an array of <strong><see cref="SharpDX.Direct3D12.VertexBufferView"/></strong> structures. </p> </dd></param>	
-        /// <include file='.\..\Documentation\CodeComments.xml' path="/comments/comment[@id='ID3D12GraphicsCommandList::IASetVertexBuffers']/*"/>	
-        /// <msdn-id>dn986883</msdn-id>	
-        /// <unmanaged>void ID3D12GraphicsCommandList::IASetVertexBuffers([In] unsigned int StartSlot,[In] unsigned int NumViews,[In] const void* pViews)</unmanaged>	
-        /// <unmanaged-short>ID3D12GraphicsCommandList::IASetVertexBuffers</unmanaged-short>	
-        public void SetVertexBuffers(int startSlot, SharpDX.Direct3D12.VertexBufferView[] descRef, int numBuffers)
+		/// <summary>	
```

Comment:
It looks like you are using tabs instead of spaces, could you fix and squash your commit?